editor/plugins/abstract_polygon_2d_editor.cpp

```diff
@@ -555,6 +574,8 @@ bool AbstractPolygon2DEditor::forward_gui_input(const Ref<InputEvent> &p_event)
 					return true;
 				}
 			}
+		} else if (k->get_keycode() == KEY_DELETE) {
+			_wip_cancel();
```

Comment:
I think delete can be outright removed. It doesn't work correctly (deletes the node) and you can just use Escape.